pkg/utils/resources/resources.go

```diff
@@ -20,10 +20,10 @@ import (
 )
 
 const (
-	NvidiaGPU = "nvidia.com/gpu"
-	AMDGPU    = "amd.com/gpu"
-	AWSNeuron = "aws.amazon.com/neuron"
-	AWSPodENI = "vpc.amazonaws.com/pod-eni"
+	NvidiaGPU v1.ResourceName = "nvidia.com/gpu"
+	AMDGPU    v1.ResourceName = "amd.com/gpu"
+	AWSNeuron v1.ResourceName = "aws.amazon.com/neuron"
```

Comment:
I don't think the AWS specific resource names should be in this core utils pkg.  Can we move to an AWS Cloud Provider utils pkg? Or maybe just on the instanceType in the AWS CP?

Reply:
Good call, moving.